packages/fxa-auth-server/lib/error.js

```diff
@@ -380,6 +382,15 @@ AppError.cannotCreatePassword = function () {
   });
 };
 
+AppError.cannotLoginNoPasswordSet = function () {
+  return new AppError({
+    code: 400,
+    error: 'Bad Request',
+    errno: ERRNO.UNABLE_TO_LOGIN_NO_PASSWORD_SET,
+    message: 'Complete account setup, please reset password to continue.',
```

Comment:
This messaging made the most sense to me, they do need to set a password to finish account setup.

Reply:
I think this gets the point across, could also be something like, "You must complete account set up by creating a password to continue" or just "You must create a password to continue." Maybe at least "create" instead of "reset" could be better since we use that verbiage elsewhere?

Reply:
Hm good point, I wonder if most users would understand "reset" password vs "create" password. Our UX mostly mentions reset password. I'll defer to vesta to make the change when she is back.